docs/user-guide.md

```diff
@@ -101,10 +101,32 @@ kedge version
 
 ## `kedge init`
 
-Initialize kedge file
+### Basic getting started
 
-```sh
-kedge init --file kedge.yml --name web --image centos/httpd --port 80
+```bash
+kedge init --name web --image centos/httpd --ports 80
+```
+
+This will create a `kedge.yml` file for a long running application running inside
+container image `centos/httpd` and exposed on port 80.
+
+### Create a different file
+
+```bash
+kedge init --file myapp.yml --name web --image centos/httpd --ports 80
+```
+
+This will create a different file named `myapp.yml` as opposed to the default
+`kedge.yml`.
+
+### Create a different controller type
+
+By default the controller type that is created is Kubernetes Deployments, which
```

Comment:
"is a Kubernetes Deployment" - also a link to Kubernetes Deployment docs would help

Reply:
done